python-checks/src/main/resources/org/sonar/l10n/py/rules/python/Sonar_way_profile.json

```diff
@@ -201,6 +202,7 @@
     "S6560",
     "S6659",
     "S6660",
+    "S6661",
```

Comment:
This rule shouldn't be in SonarWay profile. We forgot to update RSPEC

Reply:
I created a PR for this: https://github.com/SonarSource/rspec/pull/2772